pkgs/development/python-modules/virtualenv/default.nix

```diff
@@ -1,25 +1,54 @@
 { buildPythonPackage
 , fetchPypi
 , lib
-, recursivePthLoader
+, stdenv
+, pythonOlder
+, appdirs
+, contextlib2
+, distlib
+, filelock
+, importlib-metadata
+, importlib-resources
+, pathlib2
+, setuptools_scm
+, six
 }:
 
 buildPythonPackage rec {
   pname = "virtualenv";
-  version = "16.7.9";
+  version = "20.0.21";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "0d62c70883c0342d59c11d0ddac0d954d0431321a41ab20851facf2b222598f3";
+    sha256 = "1kxnxxwa25ghlkpyrxa8pi49v87b7ps2gyla7d1h6kbz9sfn45m1";
+
   };
 
-  # Doubt this is needed - FRidh 2017-07-07
-  pythonPath = [ recursivePthLoader ];
+  buildInputs = [
+    setuptools_scm
+  ];
 
-  patches = [ ./virtualenv-change-prefix.patch ];
+  propagatedBuildInputs = [
+    appdirs
+    distlib
+    filelock
+    six
+  ] ++ lib.optionals (pythonOlder "3.3") [
```

Comment:
we do not support Python < 3.5 and I think 20.09 won't even ship with 3.5 either. We do keep 2.7.

Reply:
Ah, you mean i should change to isPy2?

Reply:
changed to isPy27 for "3.3" and "3.4" in https://github.com/NixOS/nixpkgs/pull/88613/commits/118667420dc3a3677a176fe00f3586e4f7ffda28